Source/Scene/ModelExperimental/buildDrawCommands.js

```diff
@@ -75,6 +85,12 @@ export default function buildDrawCommands(
     primitiveRenderResources.boundingSphere
   );
 
+  let count = primitiveRenderResources.count;
+  if (debugWireframe) {
+    primitiveType = PrimitiveType.LINES;
+    count *= 2;
+  }
```

Comment:
Hm, this works if the primitive was `TRIANGLES`, but there are other cases that need to be handled differently:

* If the primitive was `POINTS`, should `debugWireframe` be ignored?
* `TRIANGLE_STRIP` and `TRIANGLE_FAN` have fewer indices, so it's not as simple as `count * 2`

Reply:
Looking at [This Sandcastle](http://localhost:8080/Apps/Sandcastle/index.html#c=zVhtb9s2EP4rnFE0MurISdqgm+0GW5xszZY0QW10H+JgpaVLzIYmNZJy4gb+7ztSb5TrtnlxgRlBRJF3zz13PB1PiqTQhswY3IAib4iAG9IHzdJp+MHNBaNG5O77UhjKBKhRo0XuRoIQPaGxvNEdYlQKrZFYNLsjkSGFcGtAxEEOlU1mNy8PhoyDPhI6gchIdcJumbCakWPCinlr/kTGwJFVDhqtBHDoThJBSgIRl9F1GKVKgTBDNoW6c3+mnFFxQA0EO692X+/u7ITbu69e/7LTdCAZFx2BgMq8u8VlDoYYSwGMJzzz+Lrw5CIYn3N7S7JZ+xN0Ch0yagwzCQxosaJAy1RF0CkmCIqFYRv/BnSacEDGtF0LZDtHKQft3HD4SUtRYi9aXyOhqNA8Rd+eSmSfmmgCcXH1gB9PqX2a0H9T+GHMMnjMwQdSPEm5YZuR5FKtl1vfQuoHsnkP6ATm4AfJ0ymsOZ/+ZmaS1YLcTGblgRRz99YbLEvNDYd0zB/KCAuIoSJ6MqcSpxqth1f7VDFMUWqYp7Y2jh74Awn25TSRmpmnJlqJ440exuRMMmH6XKZP3cMKyBu+/2P/0Xz60sXZ9NdQIVaSqxl4NM13Uk0p1z+CYA79aGrrKRgrqRVV5LHUDhSN5I8g5oC/RevC/dfAsfPBcywT7JBUxHCJvVncWtmXuS4lb3yuBfZFMjWhQVvXQdm1+H2PkZKPqW0HYxlhqRcmvAJzyMEO9+dHMTaFucyoYRWXsWmS8Pk+EzETV7qy0SqQm6sYWerWzulYg5rZwulrjhq5W2jRSep0rCPFxhBcpiKyZYwEIKxa3Cw2r2jcMk30Jxfo2uXF41jUY/8tNjKxV12yYZekaIjz7QryrW42q3RzXWaYKDZlhs1AhwqmcgalaDfPhwrzpyXQwq4H+mVXHeZ4GJQye7pVQptUicpSNqg0/IbYp0pjTI3isama7VrmgwnuquckVbxDcsJh8Ry1qvVswxzjw9sE0JQ9tHiW24XYopmNsg31qCIijednSk6ZLqVDMwHhbZMXpu8Eqmr6C/E8EJ+lnA5lUM2XDFr+nBeSt0gMH44zhpXoPRVXUFMmZKtVv9/cCZenTqiZhFN6G2xvbYVbZLN0eixT9+QNkgkoCBVaSnWLoFDTR/BuqsDZX1YEEiUx3oaBrjwPq0kvCDYHfeCldKx0muT583sJYnjY1cT4DP1NqvZXmzlferPzk21gl4O7etxc094hS7PO7Zi5RCzf2Pzf+ajx7C7jtSB7b8jPL+2LML4dW7hgI0kVFvoNG+bd5qhx0boHwlYNQUG8cU/N13VNqWwG3Vd5e6emPAfO5c29Ldd0OT6O9zZb04zmVHxD0z3dvvyYp1+xtDyzqN0vml6iLpYLBhaDyPYC/hGilFS1bDMTJW+Im+9WAP4RUh4Q4dLpkJfKWh3R51sX3lk7oSLmy59eBpECEIOERnA4w4L3NhMKim8bVMyozmzn+mjZHIkkNb85PzyH7OFhi2buU2b1EihWePvkrCh4+aLzcMWZlS9Xx0utXszfYbtkS0YuZs/SM38pKHYkU+MgrswE5Wv6YTadS15KRQL71YWh3FYXL71cr0tevGDebn1JZRn5nF10fWnJIeTyyitgH5/d+RqLDnl2t8KZwBdqLj4WALUDGv8tWl/b1eE8gfD48PfhP/3jo/5f/5v9LJzVE8x73Ejs4jPl73tzcnRwcHxY+dNoNXquRu8V4fmV4aueMvbgD7A/NoD9MTXYEI/T6BoreqR1EcFe21ftxWxGWPxmxfdIEnGqNa5cppwP2GcsHXu9Nsp/ocqlO3ZPZ6A4nVuxyfbecTYZhmGvjberNct+t/Sklz3sJMbmfnOMzS5K5Z1Mp/z41yqamyHcYqO+YV8m8JCYUaxnneVGvhTuU3dF+f5ESg2EklwESW447zLViozl3GM2Y4jBnbBhmkB0PZa3GB+fopuGuEPKfnqPDLJhzXNv/B8) the old Model ignored wireframe mode for `POINTS`, but `ModelExperimental` attempts to anyway.

The results do look cool though! reminds me of a yarn ball.
![image](https://user-images.githubusercontent.com/8422414/165594513-b581d284-f8df-4bf8-9456-2af35bd68286.png)